go/cmd/vtadmin/main.go

```diff
@@ -43,6 +43,8 @@ var (
 	defaultClusterConfig cluster.Config
 
 	rbacConfigPath string
+	enableRbac     bool
+	disableRbac    bool
```

Comment:
per [Go guidelines](https://zchee.github.io/golang-wiki/CodeReviewComments/#initialisms), these should be written as "{enable,disable}RBAC" (fwiw, the reason `rbacConfigPath` is written that way is to not export it, and "rBAC" would be also incorrect)